### ISSUE TITLE
More detailed documention of installation instructions

### ISSUE THREAD

**Issue by rcaracaus** [kss-node/kss-node#183]

**Comment by lkduerig** [comment]
Same here. I've gotten it installed, and I have a template (styleguide-template) set up, I'm not sure if I have the readme.md in the right place, I'm not sure if I have the rest of my files in the right place.

kss-node sass styleguide --template styleguide-template

... gives me a styleguide directory with 3 html files that look like a reasonable starting point but they have no navigation and the files are not in a directory structure. I'm sure I've got something wrong with my setup here, could you be more clear in the documentation how to solve this kind of issue?

**Comment by rupl** [comment]
I had the same troubles when I tried to use a custom template. If you just want to get started hassle-free, do not attempt to use a custom template. Unfortunately I could not isolate the root cause for the nav disappearing when using the `--template` argument, but avoiding it got me up and running.

**Comment by lkduerig** [comment]
I tried first without custom template. I got the same result, no navigation.

**Comment by rupl** [comment]
Here's  the command that worked for me (after deleting the template I started with). My `dist` folder is where all my Sass ends up compiled, in `css`, `img`, and `fonts` directories. Generating the styleguide in this `dist` folder lets me test the assets exactly how they'll be deployed.

```
node_modules/.bin/kss-node sass --css css/style.min.css --destination dist
```

The other holdup was that I was using Sass, and my `style.css` contains nothing but Sass `@import` statements. I had to go in and add KSS-compatible comments to that file or the styleguide generates no content.
